src/Libraries/RevitNodes/Elements/Views/ScheduleView.cs

```diff
@@ -162,36 +201,56 @@ public static ScheduleView CreateSchedule(Category category, string name, string
         {
             if (category == null)
             {
-                throw new System.ArgumentNullException(Properties.Resources.CategoryArgumentException);
+                throw new ArgumentNullException(Properties.Resources.CategoryArgumentException);
             }
             if (name == null)
             {
-                throw new System.ArgumentNullException(Properties.Resources.NameArgumentException);
+                throw new ArgumentNullException(Properties.Resources.NameArgumentException);
             }
             if (scheduleType == null)
             {
-                throw new System.ArgumentNullException(Properties.Resources.ScheduleTypeArgumentException);
+                throw new ArgumentNullException(Properties.Resources.ScheduleTypeArgumentException);
             }
 
-            ScheduleType t = (ScheduleType)System.Enum.Parse(typeof(ScheduleType), scheduleType);
+            var t = (ScheduleType)Enum.Parse(typeof(ScheduleType), scheduleType);
```

Comment:
hmm, I know this was fine before, but what do you think about wrapping this parse in a try catch explicitly with some kind of unsupported schedule type message?